source/5-booter.md

```diff
@@ -77,90 +77,91 @@ Designed to be filled with `plist dictionary` values, describing each patch. See
 **Failsafe**: `0`
-**Description**: Number of patch occurrences to apply. `0` applies the patch to all occurrences found.
+**Description**: 修补的次数，超过这一次数后便不再修补。`0` 表示修补所有查找到的地方。
 
 ### 4. `Enabled`
 
 **Type**: `plist boolean`
 **Failsafe**: `false`
-**Description**: This booter patch will not be used unless set to `true`.
+**Description**: 除非设置为 `true`，否则将不会应用该补丁。
 
 ### 5. `Find`
 
 **Type**: `plist data`
 **Failsafe**: Empty
-**Description**: Data to find. This must equal to `Replace` in size.
+**Description**: 要查找的数据。必须与 `Replace` 的大小相等。
 
 ### 6. `Identifier`
 
 **Type**: `plist string`
 **Failsafe**: Empty
-**Description**: `Apple` for macOS booter (generally `boot.efi`); or a name with suffix (e.g. `bootmgfw.efi`) for a specific booter; or `Any` / empty string (failsafe) to match any booter.
+**Description**: `Apple` 代表 macOS 启动器（通常是 `boot.efi`）；带有后缀的名称（如 `bootmgfw.efi`）代表特定的启动器；`Any` 或空字符串（默认）代表任何启动器。
 
 ### 7. `Limit`
 
 Type: `plist integer`
 Failsafe: `0`
-Description: Maximum number of bytes to search for. Can be set to `0` to look through the whole booter.
+Description: 搜索的最大字节数。可以设置为 `0` 来查找整个启动器。
 
 ### 8. `Mask`
 
 **Type**: `plist data`
 **Failsafe**: Empty
-**Description**: Data bitwise mask used during find comparison. Allows fuzzy search by ignoring not masked (set to zero) bits. Can be set to empty data to be ignored. Must equal to `Find` in size otherwise.
+**Description**: 在查找比较的过程中使用数据位掩码。允许通过忽略未被屏蔽的 bit（设置为 `0`）进行模糊搜索。若留空则代表忽略，否则其大小必须等于 `Find`。
 
 ### 9. `Replace`
 
 **Type**: `plist data`
 **Failsafe**: Empty
-**Description**: Replacement data of one or more bytes.
+**Description**: 一个或多个字节的替换数据。
 
 ### 10. `ReplaceMask`
 
 **Type**: `plist data`
 **Failsafe**: Empty
-**Description**: Data bitwise mask used during replacement. Allows fuzzy replacement by updating masked (set to non-zero) bits. Can be set to empty data to be ignored. Must equal to `Replace` in size otherwise.
+**Description**: 替换时使用的数据位掩码。允许通过更新掩码（设置为非 `0`）来进行模糊替换。若留空则代表忽略，否则其大小必须等于 `Replace`。
 
 ### 11. `Skip`
 
 **Type**: `plist integer`
 **Failsafe**: `0`
-**Description**: Number of found occurrences to be skipped before replacement is done.
+**Description**: 在替换前要跳过的发现事件数。
 
 ## 5.5 Quirks 属性
 
 ### 2. `AllowRelocationBlock`
 
 **Type**: `plist boolean`
 **Failsafe**: `false`
-**Description**: Allows booting macOS through a relocation block.
+**Description**: 允许通过重定位块来启动 macOS。
 
-Relocation block is a scratch buffer allocated in lower 4 GB to be used for loading the kernel and related structures by EfiBoot on firmwares where lower memory is otherwise occupied by the (assumed to be) non-runtime data. Right before kernel startup the relocation block is copied back to lower addresses. Similarly all the other addresses pointing to relocation block are also carefully adjusted. Relocation block can be used when:
+重定位块是一个分配在低 4GB 内存的缓冲区，EfiBoot 用它来加载内核和相关结构，低层内存会被非运行时数据（假定）所占用。内核启动之前，重定位块会被复制回低层内存。同样，所有指向重定位块的其他内存地址也会作出相应调整。重定位块可用于以下情况：
```

Comment:
You may want to review this section as I'm terrible at translating the term "lower memory"....

Reply:
```suggestion
重定位块（Relocation Block）是一个分配在低位 4GB 内存的缓冲区。EfiBoot 使用哦该这部分内存加载内核和相关结构，否则这部分区块会被「非运行时」数据（假定）所占用。在内核启动之前，重定位块的内容会被复制回位于低位的保留内存。同理，所有指向重定位块的其他内存地址也会作出相应调整。在下述情况中重定位块会被使用：
```